pkg/cli/admin/top/OWNERS

```diff
@@ -0,0 +1,8 @@
+# See the OWNERS docs: https://git.k8s.io/community/contributors/guide/owners.md
```

Comment:
`pkg/cli/admin/top` command exists and it is maintained by workloads team. What about moving this command to `pkg/cli/admin/toppvc`, etc. and have dedicated OWNERS?.

Reply:
It is also better to have an alias like this https://github.com/openshift/oc/blob/master/pkg/cli/admin/inspectalerts/OWNERS and https://github.com/openshift/oc/blob/e005223acd7c478bac070134c16f5533a258be12/OWNERS_ALIASES#L9

Reply:
ack